lib/src/components/NxTag/NxTag.scss

```diff
@@ -61,6 +61,13 @@
   }
 }
 
+.nx-tag__input {
```

Comment:
I thought we were going with just `appearance: none; -moz-appearance: none;` for this situation these days.  Which brings back my other point that we should really make a mixin for this, that way we can stop second-guessing the right way to do it in each situation.

Reply:
> I thought we were going with just `appearance: none; -moz-appearance: none;` for this situation these days. Which brings back my other point that we should really make a mixin for this, that way we can stop second-guessing the right way to do it in each situation.

I don't recall that decision and it doesn't work in Safari. 
<img width="300" alt="Screen Shot 2021-11-05 at 1 24 55 PM" src="https://user-images.githubusercontent.com/1991443/140552887-69954843-dd1b-405b-a59e-24100ad382c1.png">

Reply:
> I don't recall that decision and it doesn't work in Safari.

Seems line `-webkit-appearance` is also necessary.  See `TransferListHalf.scss`

Reply:
> > I don't recall that decision and it doesn't work in Safari.
> 
> Seems line `-webkit-appearance` is also necessary. See `TransferListHalf.scss`

`@mixin` created and implemented https://github.com/sonatype/sonatype-react-shared-components/pull/566/commits/eb767c8d2e953f88253039f4938c67c2fe9967b9